packages/react-tabs/src/tab.constants.ts

```diff
@@ -0,0 +1,13 @@
+// TODO: These constants should be replaced with design tokens
```

Comment:
is this TODO being punted for now?

Reply:
Yes - other controls like button has spacings that aren't defined in the design tokens, but are specified in the figma.